### ISSUE TITLE
Doesn't work on NextJS

### ISSUE THREAD

**Issue by lolcookie** [instea/react-native-popup-menu#159]
When trying to build on NextJS + RNW there is an error.

```
TypeError: (0 , _defineProperty.default) is not a function ...
```

Example repo:
https://github.com/lolcookie/react-native-web-nextjs-typescript-firebase/tree/react-native-popup
failing command:
`npm run build-app`

**Comment by sodik82** [comment]
Thanks for letting us know. RNW is supported by community. Maybe someone can help :)

Not sure what `next-transpile-modules` does but afaik popup menu is already transpiled. Do you need it for RNW and popup menu?

Just for reference, this seems to be the commit https://github.com/lolcookie/react-native-web-nextjs-typescript-firebase/commit/25a6b49766f552cb5eadaae06077066bc85f741e

**Comment by lolcookie** [comment]
@sodik82 Thanks for getting back to me, I have created another repo from the nextjs example with just react popup menu added, it fails on `npm run dev` with `{ Error: Cannot find module 'react-native'...`

https://github.com/lolcookie/react-native-popup-menu-nextjs

**Comment by sodik82** [comment]
> I found a possible solution here:
> martpie/next-transpile-modules#8 (comment)

does it works now?

**Comment by lolcookie** [comment]
@sodik82 yes but running the production build is failing with the same error, I will have to try to replicate it. I was hoping finding a solution without transpiling the modules was possible.
